Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -318,6 +318,64 @@ CommandLineArgs::CommandLineArgs(const std::vector<std::wstring>& args)
             throw hresult_invalid_argument(msg.c_str());
         }
     }
+
+    // Set up output directories
+    {
```

Comment:
Extra brackets?

Reply:
Trying to limit the scope of the filesystem path variables.  Open to removing them.

Reply:
I would prefer to have this code separated in a member function to make the code more readable. What do you think?

Reply:
That's fine, I would just be passing in the 3 strings by reference, right?

Reply:
Yep yep!